docs/databricks.md

```diff
@@ -0,0 +1,45 @@
+# Databricks
+
+## Replicate
+
+To replicate snapshot and incremental data of a TiDB Table to Databricks:
+
+```shell
+export AWS_ACCESS_KEY_ID=<ACCESS_KEY>
+export AWS_SECRET_ACCESS_KEY=<SECRET_KEY>
+
+./tidb2dw databricks \
+    --storage s3://my-demo-bucket/prefix \
+    --table <database_name>.<table_name> \
+    --databricks.host dbc-********-****.cloud.databricks.com \
+    --databricks.endpoint 2**************4 \
+    --databricks.catalog <catalog> \
+    --databricks.schema <schema> \
+    --databricks.token dapi********************************
```

Comment:
do we still need token when use long term cred?

Reply:
Yes. This token defines the auth mechanism between `tidb2dw` and the `Databricks`. The long-term credential or AWS STS is between `Databricks` and `AWS S3`. I'll update `databricks.md` later.

Reply:
Updated.